doc/Learn/Tutorials/Logging/HowTo-InternalLogging.md

```diff
@@ -5,57 +5,73 @@ uid: Learn.Tutorials.Logging.UseInternalLogging
-            .Build(enableUnoLogging: true);
-    }
+        var appBuilder = this.CreateBuilder(args)
+            .Configure(host => {
+                host
+                .UseLogging()
+            });
+
+        Host = appBuilder.Build();
+
+        // Connect Uno internal logging to the same logging provider
+        Host.ConnectUnoLogging();
+    ...
     ```
 
 ### 2. Set verbosity level of XAML logging
 
 * There are multiple log levels that correspond to differing degrees of severity:
 
-  - **Trace** : Used for parts of a method to capture a flow.
-  - **Debug** : Used for diagnostics information.
-  - **Information** : Used for general successful information. _Generally the default minimum._
-  - **Warning** : Used for anything causing application oddities. It is automatically recoverable.
-  - **Error** : Used for anything fatal to the current operation but not to the whole process. It is potentially recoverable.
-  - **Critical** : Used for anything forcing a shutdown to prevent data loss or corruption. It is not recoverable.
+| Level | Description |
+|-------|-------------|
+| Trace | Used for parts of a method to capture a flow. |
+| Debug | Used for diagnostics information. |
+| Information | Used for general successful information. Generally the default minimum. |
+| Warning | Used for anything that can potentially cause application oddities. Automatically recoverable. |
+| Error | Used for anything that is fatal to the current operation but not to the whole process. Potentially recoverable. |
+| Critical | Used for anything that is forcing a shutdown to prevent data loss or corruption. Not recoverable. |
 
-* To increase the verbosity of the events recorded when using the development hosting environment, you can tweak the minimum levels as well as those for the XAML layout.
+* To increase the verbosity of the events recorded when using the development hosting environment, you can adjust the minimum levels as well as those for the XAML layout.
 
 * Add a call for `UseLogging` to the `IHostBuilder` chain from above, and conditionally enable the recording of debug events depending on the hosting environment:
 
     ```csharp
-    public App()
+    private IHost Host { get; }
+
+    protected override void OnLaunched(LaunchActivatedEventArgs e)
     {
-        Host = UnoHost
-            .CreateDefaultBuilder()
+        var appBuilder = this.CreateBuilder(args)
+            .Configure(host => {
+                host
+                .UseLogging(configure:
+                    (context, services) =>
+                        services.SetMinimumLevel(
+                        context.HostingEnvironment.IsDevelopment() ?
+                            LogLevel.Trace :
+                            LogLevel.Error)
+                        )
+            });
+
+        Host = appBuilder.Build();
+
     #if DEBUG
-            .UseEnvironment(Environments.Development)
+        Host.UseEnvironment(Environments.Development);
```

Comment:
This needs to be done as part of the host builder, rather than after host is built
Otherwise the logging level won't be correctly set